### ISSUE TITLE
Taxonomy Term field don't show in blueprint if "display" field writing on transliterate language (russian)

### ISSUE THREAD

**Issue by itrlink** [statamic/cms#2373]
## Bug Description
Taxonomy Term field don't show in blueprint if "display" field writing on transliterate language (russian).
If writing on English - all fine.

Please check =)

**Statamic version:** 3.0.5

**PHP version:** 7.4

**Install method (choose one):**
- Fresh install from `statamic/statamic`

In blueprint:
![blueprint](https://user-images.githubusercontent.com/65292610/92334598-49ad2280-f098-11ea-8001-c8aa4134d929.jpg)

in entry page:
![entry](https://user-images.githubusercontent.com/65292610/92334605-5a5d9880-f098-11ea-8ae6-37aa155f3e80.jpg)

**Comment by itrlink** [comment]
With form field same problem =)

**Comment by ebeauchamps** [comment]
hey, we need to be solved!

**Comment by duncanmcclean** [comment]
I'm aware this one has been open for a while now. I've just tried to reproduce on the latest version of Statamic and it seems to be working fine (I used OCR to get the display name from your screenshot):

![CleanShot 2023-11-03 at 15 57 53](https://github.com/statamic/cms/assets/19637309/b17b18d2-960f-48ee-ab59-f3355d5ad2e2)

I'm going to close this issue now. If this is still causing issues for you on the latest version, let us know and we can re-open.
